Dev1-File1.txt

```diff
@@ -5,3 +5,5 @@ dev2-rebase-2
 dev1-rebase-2
 dev1-rebase-3
```

Comment:
Correct this

Reply:
I added comment as u requested and updated the fix